src/index.ts

```diff
@@ -62,7 +49,8 @@ export const release = async (): Promise<void> => {
       isNodeModule: processInputNodeModule(),
       releaseAssets: processInputReleaseAssets(),
     }),
-    releaseRules: releaseRulesExtension,
+    preset: 'angular',
```

Comment:
I know that "we" (Dott) use the 'angular' preset, but is this something that should be hardcoded? This removes the flexibility of using this release-me action with other commit messages.

Reply:
https://github.com/ridedott/release-me-action/pull/197#discussion_r440015993

We can do that in a separate PR?